src/ORM/Table.php

```diff
@@ -279,7 +279,12 @@ public function __construct(array $config = [])
             }
         }
         $this->_eventManager = $eventManager ?: new EventManager();
-        $this->_behaviors = $behaviors ?: new BehaviorRegistry($this);
+        if ($behaviors) {
+            $behaviors->setTable($this);
+        } else {
+            $behaviors = new BehaviorRegistry($this);
```

Comment:
@robertpustulka You made the `$table` argument optional for `BehaviorRegistry` constructor :)